packages/replay/src/coreHandlers/handleClick.ts

```diff
@@ -311,3 +306,47 @@ function isClickBreadcrumb(breadcrumb: Breadcrumb): breadcrumb is ClickBreadcrum
 function nowInSeconds(): number {
   return Date.now() / 1000;
 }
+
+/** Update the click detector based on a recording event of rrweb. */
+export function updateClickDetectorForRecordingEvent(clickDetector: ReplayClickDetector, event: RecordingEvent): void {
+  try {
+    // We interpret a full snapshot as a mutation (this may not be true, but there is no way for us to know)
```

Comment:
I think this is fine - we've thought about doing a full snapshot at a timed interval so lessen the effects of a corrupted segment. Though we haven't done anything with that idea.

Reply:
After talking about this, maybe we should just ignore full snapshots.
There is a _small_ possibility that this leads to a false positive (e.g. a click leads to a change that is only reflected in the full snapshot, but not in any incremental snapshot). But probably in reality the chances of this are pretty slim 🤔

Reply:
@billyvg so I changed this to ignore full snapshots, and also added integration tests for these scenarios:

1. a click triggers an error and no mutation - slow click is captured ✔️ 
2. a click triggers an error and a mutation - slow click is not captured ✔️